src/sdg_hub/blocks/llm_utils.py

```
@@ -0,0 +1,166 @@
+        Output column name(s) for parsed results.
+    start_tags : List[str], optional
+        List of start tags for tag-based parsing, by default [].
+    end_tags : List[str], optional
+        List of end tags for tag-based parsing, by default [].
+    parsing_pattern : Optional[str], optional
+        Regex pattern for custom parsing, by default None.
+    parser_cleanup_tags : Optional[List[str]], optional
+        List of tags to clean from parsed output, by default None.
+    """
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many positional arguments (8/5)

(R0917)